src/scene/shader-lib/chunks-wgsl/common/frag/envProc.js

```diff
@@ -0,0 +1,13 @@
+export default /* wgsl */`
```

Comment:
I'm not 100% keen on the `envProc` naming. `skybox`?

Reply:
we already have skybox chunk that does the skybox. This is just a small chunk used by both skybox and lit-shader